src/pages/waiting-room/waiting-room.module.ts

```diff
@@ -1,24 +1,9 @@
 import { NgModule } from '@angular/core';
-import { IonicPageModule } from 'ionic-angular';
-import { WaitingRoomPage } from './waiting-room';
-import { EffectsModule } from '@ngrx/effects';
-import { WaitingRoomAnalyticsEffects } from './waiting-room.analytics.effects';
-import { ComponentsModule } from '../../components/common/common-components.module';
-import { ReactiveFormsModule } from '@angular/forms';
-import { TranslateModule } from 'ng2-translate';
+import { CatBWaitingRoomPageModule } from './cat-b/cat-b-waiting-room.module';
```

Comment:
for the module name a naming convention of <page><cat>module eg WaitingRoomPageBModule.
With the filename waiting-room-b.module

Reply:
Also wonder if for folder structure instead of ./cat-b  it might be better to have  ./category/b/

Reply:
on the other hand it makes it easier to search and find files if they have the full name including the category